source/agora/test/ValidatorCleanRestart.d

```diff
@@ -94,7 +94,7 @@ unittest
 
         // Check enrollment
         set_b.each!(node =>
-            retryFor(node.getEnrollment(enroll.utxo_key) == enroll, 5.seconds));
+             retryFor(node.getEnrollment(enroll.utxo_key) == enroll, 5.seconds));
```

Comment:
Why the indent change ? I think we always indent by 4.

Reply:
Accidental indent change when deciding to not include a default for the retryFor. Will fix also.